src/components/ImageDetails/ReuseSurvey.vue

```diff
@@ -29,7 +33,7 @@ export default {
   computed: {
     formLink() {
       const location = this.location
-      return `https://docs.google.com/forms/d/e/1FAIpQLSeSApxNMup8Ujt-8Vjv53ngltzhJeaHspMykHCD8VKQ39yXAA/viewform?usp=pp_url&entry.1690035721=${location}`
+      return `${reuseForm}?entry.${imageLinkEntry}=${location}`
```

Comment:
What was the `usp=pp_url` used for? I assume it's not necessary anymore.

Reply:
It seems to me that the `usp` query param usually indicates [where the request came from](https://stackoverflow.com/a/41904286). In that regard, it is very similar to UTM tags. 

Anyway, it didn't have any effect on the form so I dropped it.